### PR TITLE
bpo-36264: Updates documentation for change to expanduser on Windows

### PR DESCRIPTION
**Pull request by zooba**
https://bugs.python.org/issue36264

### CODE REVIEW

**Review by asottile — APPROVED** [review]
thanks! sorry I missed this

File: Doc/whatsnew/3.8.rst [hunk 1]
```diff
@@ -236,6 +236,10 @@ now return ``False`` instead of raising :exc:`ValueError` or its subclasses
 characters or bytes unrepresentable at the OS level.
 (Contributed by Serhiy Storchaka in :issue:`33721`.)
 
+:func:`~os.path.expanduser` on Windows now prefers the :envvar:`USERPROFILE`
+environment variable and does not use :envvar:`HOME`, which is not normally set
+for regular user accounts.
+
```
Comment (zware) [inline Doc/whatsnew/3.8.rst:242]: I'm not sure this is in the right section; I'd expect it in "Changes in the Python API" rather than "Improved Modules".

Comment (zooba) [reply Doc/whatsnew/3.8.rst:242]: I mean, it clearly belongs in the section about `os.path`. I'll add it to both.

**Comment by zooba** [comment]
Just did the merge to get it done. What's New normally gets a decent review before release, so if the duplication sticks out as bad then it will be tidied up later.

**Pull request merged**